build.gradle

```diff
@@ -311,6 +311,17 @@ allprojects {
       compileOnly      group: 'org.checkerframework', name: 'checker-qual', version: checkerFrameworkVersion
       checkerFramework group: 'org.checkerframework', name: 'checker',      version: checkerFrameworkVersion
     }
+
+    if (project.name != 'qual') {
+      dependencies {
+        implementation project(':qual')
+        checkerFramework project(':qual')
+      }
+      checkerFramework {
```

Comment:
All subprojects depend on `qual`... slight overkill probably but it's modified so rarely that #yolo